xgcm/test/test_grid_ufunc.py

```diff
@@ -678,22 +813,28 @@ class GridOpsMockUp:
     """
 
     @staticmethod
-    @as_grid_ufunc(signature="(X:center)->(X:left)")
-    def diff_center_to_left(a):
```

Comment:
Do we actually want to use type hinting here? Or in our `gridops.py` module? It's arguably harder to read quickly for the simple functions we are currently storing in there.